Loggie/Classes/Alamofire/EventMonitor.swift

```diff
@@ -18,21 +18,19 @@ public final class EventMonitor: Alamofire.EventMonitor {
     public func request(_ request: Request, didGatherMetrics metrics: URLSessionTaskMetrics) {
         /// we're handling only `DataRequest`s for now
         guard let dataRequest = request as? DataRequest, let urlRequest = dataRequest.request else { return }
```

Comment:
We can combine all these guard lets into a single one, since they all just return.